scripts/jenkins/build.sh

```diff
@@ -11,18 +11,16 @@ mkdir -p $build_dir
 cd $build_dir
 
 if [ ${myhost} == "kesch" ]; then
-  BOOST_DIR="/scratch/jenkins/workspace/boost_build/boost_version/1.67.0/slave/kesch/boost_1_67_0/"
   PROTOBUFDIR="/scratch/jenkins/workspace/protobuf/slave/kesch/install/lib64/cmake/protobuf/"
 elif [ ${myhost} == "daint" ]; then
-  BOOST_DIR="/scratch/snx3000/jenkins/workspace/boost_build/boost_version/1.67.0/slave/daint/boost_1_67_0/"
   PROTOBUFDIR="/scratch/snx3000/jenkins/workspace/protobuf/slave/daint/install/lib64/cmake/protobuf/"
 else
   echo" Error Machine not found: ${myhost}"
   exit 1
 fi
```

Comment:
i think this block should be moved into `env_{host}.sh` because it is a host dependent setting.